components/brave_rewards/resources/android_page/storage.ts

```diff
@@ -47,7 +47,7 @@ export const defaultState: Rewards.State = {
     shouldAllowAdsSubdivisionTargeting: true,
     adsUIEnabled: false,
     adsIsSupported: false,
-    needsBrowserUpdateToSeeAds: false,
+    needsBrowserUpdateToViewAds: false,
```

Comment:
We should increment the "version" number in this data structure, since we are updating the schema.

Reply:
Thanks. Bumped to version 3.